Wheel_Speed_Algos.cpp

```diff
@@ -0,0 +1,91 @@
+#include <Arduino.h>
+
+#include "Wheel_Speed_Algos.h"
+
+const int STARBOARD_PIN = 2;
+const int PORT_PIN = 3;
+
+volatile unsigned int sboard_curr = 0;
+volatile unsigned int port_curr = 0;
+volatile unsigned int sboard_old = 0;
+volatile unsigned int port_old = 0;
+
+void sboard_click()
+{
+	sboard_old = sboard_curr;
+	sboard_curr = TCNT1;
+}
+
+void port_click()
+{
+	port_old = port_old;
+	port_curr= TCNT1;
+}
+
+//interrupt routine for timer2 overflow.
+//timer 2 is an 8bit timer with a 1024 scaling
+//Will run at 8MHz/256/1024 so ~30Hz
+//also am only pretty sure of 8MHz, if it's in fact 16 just some constant changing.
+
+//timer1 is at 125,000 Hz (8 MHz / 64) so one unit is 8*10^-6
+//one minute is 7,500,000 timer units
+//8 MHz / 64 * 60 = MAGIC because that's the number of timer clicks per minute
+#define MAGIC 7500000
+ISR(TIMER2_OVF_vect)
```

Comment:
As discussed offline, consider strategies to handle the case where no clicks occur in a given (1/.03) ms window.